apps/somnia/readme.md

```diff
@@ -0,0 +1,26 @@
+# somnia
+
+A discord bot for Genshin Optimizer
+
+Commands:
+
+- `/button` - Sends a button that increases a counter when pressed, used to test that bot is working as well as its response time
+- `/archive` - Queries a database of characters, weapons, and artifacts in Genshin Impact
+- `/databank` - Queries a database of characters, weapons, and artifacts in Star Rail
+
+# Running the bot
+
+Create `src/config.json` in the following format:
+
+```
+{
+  "token": "<insert-discord-bot-token>",
```

Comment:
https://discord.com/developers/docs/quick-start/getting-started#step-1-creating-an-app
Maybe just toss this link somewhere in here for a bit more direction